Marlin/src/pins/stm32f1/pins_CREALITY_V45x.h

```diff
@@ -109,3 +113,25 @@
 // Misc. Functions
 //
 #define CASE_LIGHT_PIN                      PA6
+
+#if HAS_TMC220x
+
+  //
+  // TMC2208 mode
+  //
+  //#define TMC2208_STANDALONE
+
+  #define X_HARDWARE_SERIAL  MSerial2
```

Comment:
What purpose does this serve? The stepper drivers are standalone as far as I know.